src/baseevents.cpp

```diff
@@ -74,6 +74,7 @@ bool BaseEvents::loadFromXml()
 
 		if (!success || !registerEvent(event, node)) {
 			delete event;
+			return false;
```

Comment:
Wait there, failing here just means the spell could not be loaded, not that loading the XML list failed. There's a difference between loading the XML and loading the Lua script.